packages/react/src/file-upload/FileUploadExistingFile.tsx

```diff
@@ -10,52 +10,46 @@ import { ExistingFile, formatFileSize } from './utils';
 
 type FileUploadExistingFileProps = {
 	file: ExistingFile;
-	onRemove: MouseEventHandler<HTMLButtonElement>;
 	hideThumbnails?: boolean;
+	onRemove: MouseEventHandler<HTMLButtonElement>;
 };
 
 export const FileUploadExistingFile = ({
 	file,
-	onRemove,
 	hideThumbnails,
+	onRemove,
 }: FileUploadExistingFileProps) => {
 	const { name, size, href, thumbnailSrc } = file;
 	const showThumbnail = !hideThumbnails;
 	return (
 		<Flex
-			rounded
 			as="li"
-			aria-label={`Uploaded file. ${name}`}
+			background="success"
 			gap={0.5}
 			justifyContent="space-between"
-			background="success"
+			rounded
 		>
 			<Flex>
 				{showThumbnail && <FileUploadFileThumbnail src={thumbnailSrc} />}
-				<Flex alignItems="center" paddingLeft={1} gap={0.5}>
+				<Flex alignItems="center" gap={0.5} paddingLeft={1}>
 					<Box flexShrink={0}>
 						<SuccessFilledIcon
-							color="success"
-							size="md"
 							aria-hidden="false"
 							aria-label="Success"
+							color="success"
 							css={{ display: 'block' }}
+							size="md"
 						/>
 					</Box>
 					{href ? (
-						<Text paddingY={1.5}>
-							<TextLink
-								href={href}
-								target="_blank"
-								rel="noopener noreferrer"
-								css={{ wordBreak: 'break-all' }}
-							>
+						<Text breakWords paddingY={1.5}>
+							<TextLink href={href} rel="noopener noreferrer" target="_blank">
 								{name}
 								{size ? ` (${formatFileSize(size)})` : null}
```

Comment:
Should we add the hidden text: `(opens in new window)`? Or `(opens in new tab)`?

Reply:
Hmmmm I would expect TextLink to do that automatically…